metisp/workflows/metis/metis_datasources.py

```diff
@@ -55,7 +55,7 @@
             .build())
 
 lm_raw_std = (data_source()
-            .with_classification_rule(lm_raw_std_class)
+            .with_classification_rule(lm_raw_img_std_class)
```

Comment:
Here the data source itself is still called `lm_raw_std`, which is still potentially confusing.